server/resources/js/app/widb.js

```diff
@@ -21,8 +21,16 @@ class WirelessDebug {
     /** @private @const {?MetricGrapher} */
     this.metricGrapher = null;
 
+    /** @private @const {!Object}*/
+    this.tableConfig_ = {
+      'paging': false,
+      'lengthMenu': [-1],
+      'scrollY': '75vh',
+      'scrollCollapse': true,
+    };
+
     /** @private @const {!DataTable} */
-    this.dataTable = $('#log-table').DataTable();
+    this.dataTable;
```

Comment:
Wait... How does this work? Shouldn't this be titled this.data_table? Or is it a different from the data_table on line number 63?